doc/conf.py

```diff
@@ -151,8 +150,282 @@
+      "icon": "fa-brands fa-github"
+    },
+    {
+      "name": "Twitter/X",
+      "url": "https://twitter.com/dipymri",
+      "icon": "fa-brands fa-twitter"
+    },
+    {
+      "name": "YouTube",
+      "url": "https://www.youtube.com/c/diffusionimaginginpython",
+      "icon": "fa-brands fa-youtube"
+    },
+    {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/company/dipy/",
+      "icon": "fa-brands fa-linkedin"
+    },
+  ],
+  "logo": {
+    "image_dark": "_static/images/logos/dipy-logo.png",
+    "alt_text": "DIPY",
+  },
+  "footer_start": ["components/footer-sign-up.html"],
+  "footer_signup_data": {
+    "heading": "Never miss an update from us!",
+    "sub_heading": "Don't worry! we are not going to spam you."
+  },
+  "footer_end": ["components/footer-sections.html"],
+  "footer_links": [
+    {
+      "title": "About",
+      "links": [
+        {
+          "name": "Developers",
+          "link": "https://dipy.org/team",
+          "link_type": "inter"
+        },
+        {
+          "name": "Support",
+          "link": "https://github.com/dipy/dipy/discussions",
+          "link_type": "external"
+        },
+        {
+          "name": "Download",
+          "link": "user_guide/installation"
+        },
+        {
+          "name": "Get Started",
+          "link": "user_guide/getting_started"
+        },
+        {
+          "name": "Tutorials",
+          "link": "examples_built/index"
+        },
+        {
+          "name": "Videos",
+          "link": "https://www.youtube.com/c/diffusionimaginginpython",
+          "link_type": "external"
+        },
+      ]
+    }, {
+      "title": "Friends",
+      "links": [
+        {
+          "name": "Nipy Projects",
+          "link": "http://nipy.org/",
+          "link_type": "external"
+        },
+        {
+          "name": "FURY",
+          "link": "http://fury.gl/",
+          "link_type": "external"
+        },
+        {
+          "name": "Nibabel",
+          "link": "http://nipy.org/nibabel",
+          "link_type": "external"
+        },
+        {
+          "name": "Tortoise",
+          "link": "https://tortoise.nibib.nih.gov/",
+          "link_type": "external"
+        },
+      ]
+    }, {
+      "title": "Support",
+      "links": [
+        {
+          "name": "The department of Intelligent Systems Engineering of Indiana University",
+          "link": "https://engineering.indiana.edu/",
+          "link_type": "external"
+        },
+        {
+          "name": "The National Institute of Biomedical Imaging and Bioengineering, NIH",
+          "link": "https://www.nibib.nih.gov/",
+          "link_type": "external"
+        },
+        {
+          "name": "The Gordon and Betty Moore Foundation and the Alfred P. Sloan Foundation, through the University of Washington eScience Institute Data Science Environment",
+          "link": "https://escience.washington.edu/tag/alfred-p-sloan-foundation/",
```

Comment:
I'd simplify to this:

```suggestion
          "link": "https://escience.washington.edu",
```